pkg/controller/infrastructure/infraflow/ensure.go

```diff
@@ -497,28 +461,30 @@ func (c *FlowContext) ensureFirewallRulesDeleted(ctx context.Context) error {
 	return nil
 }
 
-func (c *FlowContext) ensureKubernetesRoutesDeleted(ctx context.Context) error {
+func (fctx *FlowContext) ensureKubernetesRoutesDeleted(ctx context.Context) error {
 	log := shared.LogFromContext(ctx)
-	vpcName := c.vpcNameFromConfig()
-
-	var names []string
-	routes, err := c.computeClient.ListRoutes(ctx)
+	vpcName := fctx.vpcNameFromConfig()
+
+	routes, err := fctx.computeClient.ListRoutes(ctx, client.RouteListOpts{
+		// Filter: fmt.Sprintf("network~'%s'", vpcName),
```

Comment:
Leftover from development? If so rm. If this is here for future reference consider adding a few words as to why